volumegroup/README.md

```diff
@@ -0,0 +1,473 @@
+# CSI-Addons Operation: VolumeGroup
+
+based on deleted [3476-volume-group kep](https://github.com/kubernetes/enhancements/blob/590aa4eecb192d96e538020fc888459d9413221a/keps/sig-storage/3476-volume-group/README.md)
+
+## Container Storage Interface
+
+This section describes the interface between COs and Plugins.
+
+### RPC Interface
+
+```protobuf
+syntax = "proto3";
+package volumegroup;
+
+import "github.com/container-storage-interface/spec/lib/go/csi/csi.proto";
+import "google/protobuf/descriptor.proto";
+
+option go_package = ".;volumegroup";
+
+extend google.protobuf.MessageOptions {
+  // Indicates that this message is OPTIONAL and part of an experimental
+  // API that may be deprecated and eventually removed between minor
+  // releases.
+  bool vg_alpha_message = 1102;
```

Comment:
The alpha notations are not needed, we don't use them for other operations/messages/.. in csi-addons either. If you want to use this, use the defined values that are in `csi.proto` and drop the definitions from this file.

Leaving them out has my preference, there is currently no plan to handle _alpha_ annotation specially.

Reply:
done removed method/message alpha